files/ja/web/javascript/reference/global_objects/array/map/index.html

```diff
@@ -66,7 +66,7 @@ <h2 id="Description" name="Description">解説</h2>
 
 <h3 id="When_not_to_use_map" name="When_not_to_use_map">map() を使用すべきではない場合</h3>
 
-<p><code>map</code> は新しい配列を作成するので、返された配列を使わない場合、map を使うのはパターンに合いません。代わりに {{jsxref("Array/forEach", "forEach")}} または {{jsxref("for...of", "for-of")}} を使用してください。</p>
+<p><code>map</code> は新しい配列を作成するので、返された配列を使わない場合、map を使うのはパターンに合いません。代わりに {{jsxref("Array/forEach", "forEach")}} または {{jsxref("Statements/for...of", "for...of")}}を使用してください。</p>
```

Comment:
細かいことですが、日本語と英字の間に1文字の空白を入れるようお願いします。
```suggestion
<p><code>map</code> は新しい配列を作成するので、返された配列を使わない場合、map を使うのはパターンに合いません。代わりに {{jsxref("Array/forEach", "forEach")}} または {{jsxref("Statements/for...of", "for...of")}} を使用してください。</p>
```

Reply:
@mfuji09 
ご指摘ありがとうございます。失礼いたしました。空白を追加しましたので、ご確認のほど、よろしくお願いいたします。

※(参考)空白追加後のテスト画面で問題ないこと確認

<img width="1508" alt="スクリーンショット 2022-06-27 8 19 02" src="https://user-images.githubusercontent.com/74302915/175837739-e60bda0d-7ed4-4561-90a0-a8ccec314549.png">

Reply:
ありがとうございました。